tag-release/action.yml

```diff
@@ -38,6 +46,35 @@ runs:
         username: ${{ inputs.registry_username }}
         password: ${{ inputs.registry_token }}
 
+    - name: Wait for revision to exist in container registry
+      env:
+        EXISTING_TAG: ${{ inputs.existing_tag }}
+        MAX_TRIES: ${{ inputs.max_tries }}
+        WAIT_TIME: ${{ inputs.wait_time }}
+      id: image_wait
+      shell: bash
+      run: |
+        i=1
+        while [ true ]; do
+          echo "Checking if $EXISTING_TAG exists; try $i"
+          if docker manifest inspect $EXISTING_TAG > /dev/null; then
+            echo $EXISTING_TAG exists
+            isfound=0
```

Comment:
This is confusing since it sounds like "found it" semantics, I'd recommend `err` or `retval` since it's a Unixy exit value. Or treated like one.

Reply:
:+1: